tests/org.codechecker.eclipse.rcp.it.tests/src/org/codechecker/eclipse/plugin/utils/Utils.java

```diff
@@ -1,40 +0,0 @@
-package org.codechecker.eclipse.plugin.utils;
```

Comment:
Moved to the Shared test project because this operation is used both in unit as well as integration tests.